packages/react/src/ui-react/account/create-account-action.ts

```diff
@@ -52,18 +52,18 @@ export class CreateAccountAction extends AbstractAction<CreateAccountFormValues>
             values: initialValues,
         });
         form.param("subscriptionId", SubscriptionParameter, {
-            label: translate("subscription"),
+            label: translate("lib.common.localizer.subscription"),
```

Comment:
What if we had a `create-account-action.i18n.yml` file and added the strings to that (since the convention seems to give the yml file the same name as the Typescript file using it)? Not sure if the name might be too lengthy though